api/src/db/db.service.spec.ts

```diff
@@ -164,6 +164,59 @@ describe("DbService", () => {
         expect(docCount).toBe(8);
     });
 
+    it("can detect that a document is exactly the same as the document in the database", async () => {
+        const res: any = await service.upsertDoc({
+            _id: "group-private-content",
```

Comment:
Suggestion: import the actual document here from the JSON file so we know it's the same as the seed, and don't have to update it when the seed updates.

Reply:
I agree that this could be better, but then I have to create a function for reading a single document from file which only would be needed for this test. The current seeding functions only includes a function to read all JSON files as documents from a directory and insert/update them in the database.